checkov/terraform/context_parsers/base_parser.py

```diff
@@ -91,8 +93,10 @@ def _collect_skip_comments(self, definition_blocks: List[Dict[str, Any]]) -> Dic
         """
         Collects checkov skip comments to all definition blocks
         :param definition_blocks: parsed definition blocks
-        :return: context enriched with with skipped checks per skipped entity
+        :return: context enriched with skipped checks per skipped entity
         """
+        should_allow_multi_checks_skip = bool(os.getenv('CHECKOV_ALLOW_SKIP_MULTIPLE_ONE_LINE', 'False'))
```

Comment:
```suggestion
        should_allow_multi_checks_skip = strtobool(os.getenv('CHECKOV_ALLOW_SKIP_MULTIPLE_ONE_LINE', 'False'))
```
bool('False') equal to True.